src/test/java/com/banzaicloud/metrics/prometheus/client/exporter/TextFormatWithTimestampSuite.java

```diff
@@ -1,162 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.banzaicloud.metrics.prometheus.client.exporter;
-
-
-import com.google.common.collect.Lists;
-import io.prometheus.client.Collector;
-import org.junit.Assert;
-import org.junit.Test;
-
-import java.util.Collections;
-import java.util.List;
-
-
-public class TextFormatWithTimestampSuite {
```

Comment:
We still need this test suite to verify that the generated payload passed to PushGateway is in the correct format and contains the timestamp. (there are older and custom push gateway versions that support timestamp)

Reply:
I'll try to add that